src/main/java/uk/gov/companieshouse/acsp/controllers/AcspController.java

```diff
@@ -48,4 +47,16 @@ public ResponseEntity<Object> getAcspData(@PathVariable(TRANSACTION_ID_KEY) Stri
         }
     }
 
+    @GetMapping("/acsp-api/user/{id}/application")
+    public ResponseEntity<Object> checkHasApplication(@PathVariable("id") String id,
+                                                @RequestHeader(value = ERIC_ACCESS_TOKEN) String requestId){
+        LOGGER.info("received request to check for user applications");
+        var acspData = acspService.getAcsp(id);
```

Comment:
Because there can be more than one application for a user now... so it will be better to check the count rather than doing a get. getAcsp will have to change in some time where there will be a another unique id for the application and we would not have this unique id at this time.